src/amo/components/App/styles.scss

```diff
@@ -1,10 +1,6 @@
 @import '~normalize.css';
```

Comment:
At some point we need to fix this.

```
[0] DEPRECATION WARNING on line 1, column 8 of stdin:
[0] Including .css files with @import is non-standard behaviour which will be removed in future versions of LibSass.
```

Reply:
ah, interesting.

Reply:
more info at: https://github.com/sass/node-sass/issues/2362#issuecomment-388634848